README.md

```diff
@@ -7,28 +7,36 @@
 
 ## What is Kadalu ?
 
-[Kadalu](https://kadalu.io) is a project which started as an idea to make glusterfs's deployment and management simpler in kubernetes. The project contains operator to deploy CSI pods, and gluster storage nodes. All of gluster management is done natively in kubernetes without glusterfs's `glusterd` and `gluster` CLI tools.
+[Kadalu](https://kadalu.io) is a project to provide Persistent Storage in container ecosystem (like kubernetes, openshift, RKE, etc etc). Kadalu operator deploys CSI pods, and **gluster storage** pods as per the config. You would get your PVs served through APIs implemented in CSI.
 
-Try it in few minutes to understand more!
+## Get Started
 
-## Documentation
+Getting started is made easy to copy paste the below commands.
 
-Start with our [Quick Start Guide](doc/quick-start.adoc). More documentation is at [`doc/` folder](doc/).
+[source,console]
```

Comment:
I think either we need to change README.md to README.adoc or replace adoc variants in this file with markdown

Reply:
Ack!